users/forms.py

```diff
@@ -10,4 +10,14 @@ class SignUpForm(UserCreationForm):
 
     class Meta:
         model = User
-        fields = ('username', 'first_name', 'last_name', 'email', 'password1', 'password2', )
+        fields = ('username', 'first_name', 'last_name', 'email', 'password1', 'password2')
+
+
+class CustomPasswordReset(PasswordResetForm):
+    def clean(self):
+        email = self.cleaned_data.get("email")
+        try:
+            User.objects.get(email=email)
```

Comment:
This won't handle the following case. Consider a user has an account with email `test@test.com` and in forgot password for it he puts Test@tets.com it will show `No Account with this email Address`. Email and usernames are case insensitive. Handle it and write a test case for it.

Reply:
Fixed